src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationMergeEventListenerIntTest.kt

```diff
@@ -335,16 +334,14 @@ class ProbationMergeEventListenerIntTest : MessagingMultiNodeTestBase() {
       ),
     )
 
-    probationMergeEventsQueue!!.sqsClient.purgeQueue(
```

Comment:
As the queues dlqMaxReceiveCount were set to 3 purging the queue after the first would cause the SQS error locally. However, still unsure of why this occurred in prod